Assets/HoloToolkit/Utilities/Scripts/WorldAnchorManager.cs

```diff
@@ -136,6 +137,36 @@ public void RemoveAnchor(GameObject gameObjectToUnanchor)
         }
 
         /// <summary>
+        /// Removes all anchors from the scene and deletes them from the anchor store.
+        /// </summary>
+        public void RemoveAllAnchors()
+        {
+            // This case is unexpected, but just in case.
+            if (AnchorStore == null)
+            {
+                Debug.LogError("remove all anchors called before anchor store is ready.");
+            }
+
+            WorldAnchor[] anchors = FindObjectsOfType<WorldAnchor>();
```

Comment:
I wonder what the trade off of keeping a list of anchors would be vs searching the scene for all the `WorldAnchors`. Then we wouldn't need to reference the spatial mapping anchors because we'd only know about the ones we've been trying to keep track of.